articles/longest-common-prefix.md

```diff
@@ -0,0 +1,503 @@
+## 1. Iteration - I
```

Comment:
minor: for some of the solution titles i think we can be more descriptive. For example the LC editorial mentions horizontal scanning vs vertical scanning: https://leetcode.com/problems/longest-common-prefix/editorial/

Reply:
I will modify that in next PR.